saleor/graphql/product/types/products.py

```diff
@@ -537,6 +537,10 @@ def resolve_category(root: ChannelContext[models.Product], info):
             return None
         return CategoryByIdLoader(info.context).load(category_id)
 
+    @staticmethod
+    def resolve_description_json(root: ChannelContext[models.Product], info):
+        return root.node.description
```

Comment:
I think we should add tests or extend the existing ones to check fetching `descriptionJson` on `Product`, `Collection`, and `Category`.